Gemfile

```diff
@@ -6,15 +6,3 @@ gemspec
 # Load developer specific Gemfile
 dev_gemfile = File.expand_path("Gemfile.dev.rb", __dir__)
 eval_gemfile(dev_gemfile) if File.exist?(dev_gemfile)
-
-minimum_version =
-  case ENV['TEST_RAILS_VERSION']
-  when "6.0"
-    "~>6.0.4"
-  when "7.0"
-    "~>7.0.8"
-  else
-    "~>6.1.4"
-  end
-
-gem "activesupport", minimum_version
```

Comment:
If all of rails is going away, then we should also remove it from the test matrix: https://github.com/ManageIQ/manageiq-api-client/blob/d91929a7a2cc2b3b6a4ff757ae1232a3638b2ef9/.github/workflows/ci.yaml#L18-L21